proto/metrics/metrics.proto

```diff
@@ -710,3 +712,46 @@ message FlowlogCounters {
   // The number of failed attempts to publish a flow record.
   uint64 failed_attempts = 3;
 }
+
+// Resources used as reported by kubernetes api for the node objects.
+message KubeNodeMetrics {
+  // Name of the node
+  string name = 1;
+
+  // CPU cores in milli cores
+  uint32 cpu_cores = 2;
+
+  // CPU usage in percentage
+  float cpu_usage = 3;
+
+  // Memory usage in bytes
+  uint32 memory_usage = 4;
+
+  // Memory usage in percentage
+  float memory_usage_percentage = 5;
+}
+
+// Resources used as reported by kubernetes api for pods running eve user apps.
+message KubeEVEAppPodMetrics {
```

Comment:
Why only pod metrics ? What about the VMI ? Do we still use pod metrics for VMI too ?

Reply:
Each vmi app has an associated pod (virt-launcher-*), vmi and vmirs objects in the eve-kube-app namespace.  

The vmi lists cpu cores, memory Gi resources,  and storage bytes requested.

The vmirs list cpu cores and memory Gi resources. 

The virt-launcher pod lists two containers virt-launcher-monitor (the vm) and virt-tail (logs).  Neither of these specifically list cores but instead a Millicore value which doesn't seem to match the core count.  My virt-launcher reports 200millicores which is 1/5th of a core instead of the vmi and vmirs which report 2 cores.  Using memory requests from the virt-launcher pod does seem to be the most complete view of all memory used by all containers associated with a running domain.